vuepress/docs/.vuepress/next.js

```diff
@@ -1,69 +1,129 @@
-                path: path,
-            },
             {
                 title: 'Getting Started',
                 children: [
                     {
-                        title: 'Overview',
-                        path: path  + 'getting-started/',
+                        title: 'Introduction',
+                        path: path,
                     },
                     {
-                        title: 'Welcome Wizard',
-                        path: path + 'getting-started/welcome-wizard.md'
+                        title: 'Learn About Entando',
+                        path: path + 'getting-started/',
+                    },
+                    {
+                        title: 'Concepts Overview',
+                        path: path + 'getting-started/concepts-overview.md',
                     },
                 ]
             },
             {
-                title: 'Concepts',
+                title: 'Create Components',
                 children: [
                     {
-                        title: 'Overview',
-                        path: path + 'concepts/'
+                        title: 'Entando Component Generator',
+                        path: path + 'create-components/component-gen-overview.md',
                     },
                     {
-                        title: 'Entando Operator',
-                        path: path + 'concepts/operator-intro.md'
+                        title: 'Entando Blueprint Features',
+                        path: path + 'create-components/blueprint-features.md',
                     },
                     {
-                        title: 'Custom Resources',
-                        path: path + 'concepts/custom-resources.md'
+                        title: 'Component Generation Technologies',
+                        path: path + 'create-components/component-gen-tech.md',
                     },
                     {
-                        title: 'Accessibility',
-                        path: path + 'concepts/accessibility.md'
+                        title: 'Customizing The Component Generator',
+                        path: path + 'create-components/component-gen-customize.md',
+                    },
+                ]
+            },
+            {
+                title: 'Curate and Share Components',
+                children: [    
+                    {
+                        title: 'Entando Hub',
+                        path: path  + 'curate-share/hub.md'
+                    },
+                    {
+                        title: 'Bundle and Component Descriptors',
+                        path: path  + 'curate-share/ecr-bundle-details.md'
                     },
                     {
-                        title: 'PDA Architecture',
-                        path: path + 'concepts/pda-architecture.md'
+                        title: 'Filtering Bundles',
+                        path: path  + 'curate-share/ecr-bundle-filters.md'
```

Comment:
I think we can keep the paths shorter. They should be human readable but shorter is better for finding them later - and less likely to go completely out of date if we change the full title later. Here I would shorten to curate-share to curate.  Remember, these are visible in the URL so savvy users do notice them.